lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
Is not a problem to remove it, if we agree is a functionality we don't want to offer anymore. If we remove this, it will actually remove the ability to run test in released packages, I just wander is we're all good for that.